src/lineup/ALineUpView2.ts

```diff
@@ -80,7 +80,7 @@ export abstract class ALineUpView2 extends AView {
   protected idAccessor = (d) => d._id;
 
 
-  constructor(context: IViewContext, protected selection: ISelection, parent: Element, private options?: {}) {
+  constructor(context: IViewContext, protected selection: ISelection, parent: Element, private options?: {}, private readonly disableAddingColumns?: boolean) {
```

Comment:
please set the default to `false` such that we either have `true` or `false` not `true` and `undefined`